tests/postgres_tests/test_indexes.py

```diff
@@ -30,7 +30,6 @@ def test_deconstruction_no_customization(self):
         self.assertEqual(kwargs, {'fields': ['title'], 'name': 'test_title_%s' % self.index_class.suffix})
 
 
-@skipUnlessDBFeature('has_brin_index_support')
```

Comment:
No test method actually hit `connection.features.has_brin_index_support` below.

Reply:
BRIN indexes were introduced in PostgreSQL 9.5 and Django 2.2 will still support 9.4 so not having this will cause a failure for anyone running the test suite against 9.4 instead of 9.5+.

Reply:
@pope1ni I should have made my comment clearer; no test methods of `BrinIndexTests` are dependant of the feature flag no there's no point in skipping them if it's not available.

1. `test_suffix` simple attribute check.
2. `test_deconstruction` the feature flag is not relied upon during deconstruction.
3. `test_invalid_pages_per_range` simple `ValueError` on initialization.

In other words, this test class either relies on database behavior and should subclass `TestCase` or the feature flag which relies on version detection is not required in the first place. I opted to remove the database feature detection based test skipping since it allows us to avoid doing unnecessary transaction wrapping.

Reply:
Thanks for the explanation, makes sense.